aiohttp/web.py

```diff
@@ -133,6 +135,8 @@ def _set_loop(self, loop):
         if self._debug is ...:
             self._debug = loop.get_debug()
 
+        self._jobs = JobRunner(loop=loop)
```

Comment:
Maybe later if people will request it.
Easier to add a new `.run_sync()` method than inviting to reimplement the whole logic.
`.run_sync(func, *args, **kwargs)` should have a different signature than `.run(coro)`, it's not just a replacement.
Cancellation of synchronous jobs is also problematic challenge.